src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/ActivatorUtilities.cs

```diff
@@ -1116,6 +1222,20 @@ public static void ClearCache(Type[]? _)
                 }
             }
         }
+
+        [StructLayout(LayoutKind.Sequential)]
+        private ref struct StackAllocatedObjects
+        {
+            public const int MaxStackAllocArgCount = 8;
+            public object? _args;
```

Comment:
Can this use InlineArray?

Reply:
Yes; see recent commit. However, it is a little kludgy since [InlineArray] can't be used on a `ref struct` directly. Not sure why that is the case.